theme/static/js/hs-vue/relevant-tools.js

```diff
@@ -0,0 +1,136 @@
+/**
+* Created by Mauriel on 5/19/2019.
+*/
+
+let relevantToolsApp = new Vue({
+    el: '#apps-dropdown',
+    delimiters: ['${', '}'],
+    data: {
+        tools: [],
+        openWithTools: [],
+        resId: SHORT_ID,
+        isLoading: true,
+        trackingAppLaunchUrl: TRACKING_APPLAUNCH_URL,
+    },
+    methods: {
+        // Returns the Url needed to launch the app with this resource
+        getResAppUrl: function (tool) {
+            return this.trackingAppLaunchUrl + '?url=' + tool.url + ';name=' +  tool.title +
+                ';tool_res_id=' + tool.res_id + ';res_id=' + this.resId;
+        },
+        // Returns the Url needed to launch a file in this resource
+        getFileAppUrl: function (tool) {
+            if (tool.hasOwnProperty('file_extensions') && tool.url_file) {
+                return this.trackingAppLaunchUrl + '?url=' + tool.url_file + ';name=' + tool.title +
+                    ';tool_res_id=' + tool.res_id + ';res_id=' + this.resId;
+            }
+            return null;    // default
+        },
+        // Returns the Url needed to launch a file in this resource
+        getAggregationAppUrl: function (tool) {
+            if (tool.hasOwnProperty('agg_types') && tool.url_aggregation) {
+                return this.trackingAppLaunchUrl + '?url=' + tool.url_aggregation + ';name=' + tool.title +
+                    ';tool_res_id=' + tool.res_id + ';res_id=' + this.resId;
+            }
+            return null;    // default
+        }
+    },
+    mounted: function () {
+        let vue = this;
+        $.get("/hsapi/_internal/" + this.resId + "/relevant-tools/", function (response) {
```

Comment:
I think the name `vue` is appropriate since it is not a reserved keyword and also encapsulates well that the object is meant to reference the Vue instance.